ExpectedFailure() { return { "BadUsingTarget", 135, 138 }; }
type t {
    x := int;
    type(i := int64) : x(i) {}
}
using arg := t(5);
Main() {
    return arg.x == 5;
}